includes/tracks/events/class-wc-products-tracking.php

```diff
@@ -38,6 +39,18 @@ public function track_product_updated( $product_id, $post ) {
 		);
 
 		WC_Tracks::record_event( 'product_edit', $properties );
+
+		$product = new WC_Product( $product_id );
+		$update_properties = array(
+			'product_id'            => $product_id,
+			// 'product_type'          => TODO where does this come from?? there is no product type, only category ids.
+			'category_ids'          => $product->get_category_ids(),
+			'manage_stock'          => 0 == $product->get_manage_stock() ? 'N' : 'Y',
+			// 'stock_quantity_update' => 'Y'	TODO this may require hooking in to product edit at a lower level, which doesn't seem possible for an extension.
```

Comment:
@timmyc I don't think I'm working at the right level, can you hook in to a product edit to get the previous state?

Reply:
Sorry late to the party on this comment, wondering if this hook might be what we need to see the args being modified: https://codex.wordpress.org/Plugin_API/Filter_Reference/wp_insert_post_data